lib/repl.js

```diff
@@ -852,6 +852,57 @@ ArrayStream.prototype.resume = function() {};
 ArrayStream.prototype.write = function() {};
 
 const requireRE = /\brequire\s*\(['"](([\w@./-]+\/)?(?:[\w@./-]*))/;
+
+
+const fsTabbableMethods = [
+  'access',
+  'accessSync',
+  'chmod',
+  'chmodSync',
+  'chown',
+  'chownSync',
+  'createReadStream',
+  'createWriteStream',
+  'exists',
+  'existsSync',
+  'lchmod',
+  'lchmodSync',
+  'lchown',
+  'lchownSync',
+  'lstat',
+  'lstatSync',
+  'mkdir',
+  'mkdirSync',
+  'open',
+  'openSync',
+  'readdir',
+  'readdirSync',
+  'readFile',
+  'readFileSync',
+  'readlink',
+  'readlinkSync',
+  'realpath',
+  'realpath',
+  'realpathSync',
+  'realpathSync',
+  'rmdir',
+  'rmdirSync',
+  'stat',
+  'statSync',
+  'symlink',
+  'symlinkSync',
+  'truncate',
+  'truncateSync',
+  'unlink',
+  'unlinkSync',
+  'utimes',
+  'utimesSync',
+].join('|');
+
+const fsRE = new RegExp(
+  `fs\\.(?:${fsTabbableMethods})\\(["'](([\\w@./-]+\\/)?(?:[\\w@./-]*))`
```

Comment:
It's better to add a `\b` at the beginning of the RegExp, otherwise it will match something like `xxxxfs. readFile("`